core-bundle/src/Resources/contao/themes/flexible/basic.css

```diff
@@ -366,11 +363,6 @@ select[multiple] {
 .tl_select[multiple],.tl_mselect[multiple],.tl_select_column[multiple],.tl_select_unit[multiple],.tl_select_interval[multiple] {
 	background-image:none;
 }
-@-moz-document url-prefix() {
-	.tl_select,.tl_mselect,.tl_select_column,.tl_select_unit,.tl_select_interval {
-		padding:4px 18px 4px 2px;
```

Comment:
For Firefox on Mac OS, we need to keep the `padding-left:2px`. The other paddings do not need to be changed.